src/electionguard/tally.py

```diff
@@ -90,13 +90,24 @@ class CiphertextTallyContest(ElectionObjectBase):
     A collection of CiphertextTallySelection mapped by SelectionDescription.object_id
     """
 
+    # TODO: rename accumulate ballot
```

Comment:
TODO: fiox or remove this

Reply:
Yeah. I'd be remiss if I didn't say we need to remove this or give it an issue.